pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -393,6 +388,14 @@ func (r *ReconcileIntrusionDetection) Reconcile(ctx context.Context, request rec
 		return reconcile.Result{}, err
 	}
 
+	if isManagedCluster {
+		if esClusterConfig.ClusterName() == render.DefaultElasticsearchClusterName {
+			err = fmt.Errorf("Elasticsearch cluster name must be non-default value in managed clusters")
+			r.status.SetDegraded(operatorv1.InvalidConfigurationError, "", err, reqLogger)
```

Comment:
This is an interesting new assertion - any background on this one? It seems safe to me, but curious what prompted you to add it!

I do think we need a user-facing message in the SetDegraded call instead of `""` though.

Also, this isnt' the Elasticsearch cluster name. Rather this is the just the name of this managed cluster (which is primarily used by our elastic log ingestion pipeline).